src/components/editor/Bindings/Selector.tsx

```diff
@@ -1,147 +1,40 @@
-import type { GridRenderCellParams } from '@mui/x-data-grid';
 import type { ReactNode } from 'react';
 
-import { Box } from '@mui/material';
+import { Stack } from '@mui/material';
 
-import { deleteDraftSpecsByCatalogName } from 'src/api/draftSpecs';
 import CollectionSelectorList from 'src/components/collection/Selector/List';
-import {
-    COLLECTION_SELECTOR_NAME_COL,
-    COLLECTION_SELECTOR_UUID_COL,
-} from 'src/components/collection/Selector/List/shared';
-import BindingsSelectorName from 'src/components/editor/Bindings/Row/Name';
-import BindingsSelectorRemove from 'src/components/editor/Bindings/Row/Remove';
-import BindingsSelectorToggle from 'src/components/editor/Bindings/Row/Toggle';
 import BindingSearch from 'src/components/editor/Bindings/Search';
-import { useEditorStore_persistedDraftId } from 'src/components/editor/Store/hooks';
-import { useEntityType } from 'src/context/EntityContext';
-import { useEntityWorkflow } from 'src/context/Workflow';
-import {
-    useBinding_collections,
-    useBinding_discoveredCollections,
-    useBinding_removeBindings,
-    useBinding_setCurrentBinding,
-    useBinding_toggleDisable,
-} from 'src/stores/Binding/hooks';
-import { useBindingStore } from 'src/stores/Binding/Store';
-import { useDetailsFormStore } from 'src/stores/DetailsForm/Store';
+import { useBinding_setCurrentBinding } from 'src/stores/Binding/hooks';
 import { useFormStateStore_isActive } from 'src/stores/FormState/hooks';
-import { hasLength } from 'src/utils/misc-utils';
 
 interface BindingSelectorProps {
     disableSelect?: boolean;
-    height?: number | string;
+    legacyTransformHeightHack?: number;
```

Comment:
I really want to remove this property so marking it as something we want to remove.